packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -135,6 +138,7 @@ export const registerScaledPriceAuthority = async (
       priceAuthorityAdmin,
       priceAuthority,
     },
+    instance: { produce: produceInstance },
```

Comment:
comment / security concern: small increase in authority - authority to produce/replace any agoricNames.instance. But since we can't know `issuerName` in advance, this _is_ the least authority we can grant within the expressiveness of our permits.